webapp/app/.server/db/schema.ts

```diff
@@ -2,6 +2,10 @@ import { bytea } from "@/.server/db/custom-types";
 import { index, json, pgTable, serial, text, timestamp, unique } from "drizzle-orm/pg-core";
 import { z } from "zod";
 
+export const proposalType = z.enum(["routine", "emergency"]);
```

Comment:
ok, will do that instead

Reply:
have kept it as `proposal` table as its nicer to see and say.

lets keep columns as `calldata` and `Calls` since that has mapping with original smart contracts